covid_model/model_fit.py

```diff
@@ -56,9 +56,16 @@ def run(self, engine, method='curve_fit', window_size=14, look_back=None,
         end_t = self.actual_hosp.index.max() + 1
         end_date = self.base_specs.start_date + dt.timedelta(end_t)
 
+        # create base model
+        base_model = CovidModel(from_specs=self.base_specs, end_date=end_date)
+
+        # calculate and set new I0
+        hosp_rate = base_model.model_params['hosp']['value']['40-64'][0]  # Take first compartment's hosp rate
+        I0 = max(2.2, self.actual_hosp[0] / hosp_rate)
+        base_model.model_params['initial_seed']['value'][0] = I0
```

Comment:
In the new approach to the initial seed, we just have to change the value in model_params. Note that I didn't just use base_model.set_param() here, because we need it to be stored on the specification in model_params (which gets written to the database), not just in the param lookup dictionary (which doesn't get written to the database.